HPXMLtoOpenStudio/resources/HPXMLvalidator.xml

```diff
@@ -454,7 +454,7 @@
       <sch:assert role='ERROR' test='h:Location[text()="attic" or text()="attic - conditioned" or text()="attic - unconditioned" or text()="attic - unvented" or text()="attic - vented" or text()="basement" or text()="basement - conditioned" or text()="basement - unconditioned" or text()="crawlspace" or text()="crawlspace - conditioned" or text()="crawlspace - unconditioned" or text()="crawlspace - unvented" or text()="crawlspace - vented" or text()="exterior wall" or text()="garage" or text()="garage - conditioned" or text()="garage - unconditioned" or text()="living space" or text()="other heated space" or text()="other housing unit" or text()="other multifamily buffer space" or text()="other non-freezing space" or text()="outside" or text()="roof deck" or text()="unconditioned space"] or not(h:Location)'>Expected Location to be 'attic' or 'attic - conditioned' or 'attic - unconditioned' or 'attic - unvented' or 'attic - vented' or 'basement' or 'basement - conditioned' or 'basement - unconditioned' or 'crawlspace' or 'crawlspace - conditioned' or 'crawlspace - unconditioned' or 'crawlspace - unvented' or 'crawlspace - vented' or 'exterior wall' or 'garage' or 'garage - conditioned' or 'garage - unconditioned' or 'living space' or 'other heated space' or 'other housing unit' or 'other multifamily buffer space' or 'other non-freezing space' or 'outside' or 'roof deck' or 'unconditioned space'</sch:assert>
       <sch:assert role='ERROR' test='h:BatteryType[text()="Li-ion" or text()="lead acid" or text()="Ni-Cd" or text()="salt water" or text()="other"] or not(h:BatteryType)'>Expected BatteryType to be 'Li-ion' or 'lead acid' or 'Ni-Cd' or 'salt water' or 'other'</sch:assert>
       <sch:assert role='ERROR' test='number(h:RatedPowerOutput) &gt;= 0 or not(h:RatedPowerOutput)'>Expected RatedPowerOutput to be greater than or equal to 0</sch:assert>
-      <sch:assert role='ERROR' test='count(h:SystemIdentifier[@id]) = count(h:SystemIdentifier)'>Expected id attribute for SystemIdentifier</sch:assert>
+      <sch:assert role='ERROR' test='count(h:SystemIdentifier[@id]) = 1'>Expected SystemIdentifier with id attribute</sch:assert>

```

Comment:
@shorowit Here is that change to `HPXMLvalidator.xml` I was talking about before. This occurred when running `update_hpxmls`.